cumulus/parachains/runtimes/assets/asset-hub-westend/src/lib.rs

```diff
@@ -1092,6 +1092,7 @@ impl pallet_revive::Config for Runtime {
 	type NativeToEthRatio = ConstU32<1_000_000>; // 10^(18 - 12) Eth is 10^18, Native is 10^12.
 	type EthGasEncoder = ();
 	type FindAuthor = <Runtime as pallet_authorship::Config>::FindAuthor;
+	type CheckingAccount = xcm_config::CheckingAccount;
```

Comment:
let's use a different checking account, dedicated to erc20s, and keep this one dedicated to teleports (of any type of asset)